src/syscheckd/create_db.c

```diff
@@ -176,7 +180,9 @@ time_t fim_scan() {
     fim_registry_scan();
 #endif
     if (syscheck.file_limit_enabled) {
+        /* DEPRECATED CODE
         nodes_count = fim_db_get_count_entries(syscheck.database);
```

Comment:
Probably, we will need to reimplement this function in order to keep the feature of sending an alert with the DB status.